### PR TITLE
[fix] Fetch B2B limit from GST Settings

### PR DESCRIPTION
**Pull request by vishdha**

### CODE REVIEW

**Comment by saurabh-bhosale** [comment]
Fix for #12784

**Comment by rmehta** [comment]
default value?

File: erpnext/regional/doctype/gst_settings/gst_settings.json [hunk 1]
```diff
@@ -160,6 +160,36 @@
...
+   "fieldname": "b2b_limit", 
+   "fieldtype": "Data", 
+   "hidden": 0, 
+   "ignore_user_permissions": 0, 
+   "ignore_xss_filter": 0, 
+   "in_filter": 0, 
+   "in_global_search": 0, 
+   "in_list_view": 0, 
+   "in_standard_filter": 0, 
+   "label": "B2B Limit",
```
Comment (nabinhait) [inline erpnext/regional/doctype/gst_settings/gst_settings.json]: This limit is not for B2B, it's for B2C. Also set a better descriptive label for this.

**Review by nabinhait — CHANGES_REQUESTED** [review]

**Comment by nabinhait** [comment]
Closing this for now, please reopen after fix.

**Pull request closed without merging**